quadriga/src/main/java/edu/asu/diging/quadriga/core/data/IdGenerator.java

```diff
@@ -0,0 +1,33 @@
+package edu.asu.diging.quadriga.core.data;
```

Comment:
let's put the relational database classes/interfaces into a subpackage `sql` or something like that.

Reply:
Made this change in story Q20-16.

Reply:
Then this should not be merged like this.

Reply:
Resolved.

Reply:
where?